remoulade/state/backend.py

```diff
@@ -0,0 +1,82 @@
+from collections import namedtuple
+from enum import Enum
+
+from ..encoder import Encoder
+from .errors import InvalidStateError
+
+
+class StatesEnum(Enum):
```

Comment:
Sorry but i think we can do better name for this: as the attribute of the state is name we could name it  `StateNamesEnum` ? would be more clear right ?

Reply:
the clear the better, thanks